### DISCUSSION TITLE
Заполнение полей через отдельную форму.

### DISCUSSION THREAD

**Discussion by EKhorikov** [totumonline/totum-ru-issues-and-discussions#84] [Как это сделать?] [answered]
Добрый день.
Есть Поля в расчётной таблице циклов, которые необходимо заполнять через отдельную форму (временная таблица) по нажатию кнопки "ЗАПЛАНИРОВАТЬ" (с ограничением доступа)
![image](https://user-images.githubusercontent.com/92679628/155703070-5fccb07e-af12-4f54-b5f9-c9f827e1d79f.png)
Есть временная таблица с этими полями, которая открывается при нажатии кнопки "ЗАПЛАНИРОВАТЬ", с передачей в неё № цикла.

Код действия кнопки "ЗАПЛАНИРОВАТЬ":

= : linkToDataTable(table: 'zaplanirovat_zayavku'; title: $str; width: "60vh"; height: "60vh"; params: $row; refresh: true)

str: strAdd(str: "Планирование заявки "; str: $#nci)

row: rowCreate(field: "h_nomer_zayavki" = $#nci)

![image](https://user-images.githubusercontent.com/92679628/155703801-7540001a-b4e3-4530-bf58-c1a3fbc2e01c.png)

Код действия кнопки "запланировать" во временной таблице:

= : set (table: 'zayavka'; cycle: #h_nomer_zayavki; field: 'h_ispolnitel' = $isp; field: 'h_nachalo' = $nach; field: 'h_konec' = $kon; log: true)

isp: listSection(list: $selrow; item: "h_ispolnitel")
nach: listSection(list: $selrow; item: "h_nachalo")
kon: listSection(list: $selrow; item: "h_konec")

selrow: selectRowList(table: $#ntn; field: 'h_ispolnitel'; field: 'h_nachalo'; field: 'h_konec')

Проблема вот в чём - При выполнении кода, код данные не передаёт, поля остаются пустыми, и появляется сообщение "Ошибка выполнения операции "

**Comment by EKhorikov** [comment, marked as answer]
Перемудрил я с set.
Правильный код:
= : set (table: 'zayavka'; cycle: #h_nomer_zayavki; field: 'h_ispolnitel' = #h_ispolnitel; field: 'h_nachalo' = #h_nachalo; field: 'h_konec' = #h_konec; log: true)

Спасибо за помощь!

**Reply by ttm-alexeypolunin** [reply]
Ошибку там сделают человеческую — это артефакты одного из последних изменений.
